### DISCUSSION TITLE
Expired certificate for regolith-desktop.org?

### DISCUSSION THREAD

**Discussion by samstokes** [regolith-linux/regolith-desktop#740] [General]
Hi, I'm getting errors that the SSL cert for regolith-desktop.org has expired - is this something already known? e.g. Firefox says:
> The certificate for regolith-desktop.org expired on 10/8/2022.

I ran into this when trying to install an extra Regolith package via `apt`, but got errors like these:

> E: Failed to fetch https://regolith-desktop.org/release-ubuntu-jammy-amd64/pool/main/i/i3-swap-focus/i3-swap-focus_0.4.3-1regolith_amd64.deb  Certificate verification failed: The certificate is NOT trusted. The certificate chain uses expired certificate.  Could not handshake: Error in the certificate verification. [IP: 104.238.213.207 443]

I get a similar error from `wget https://regolith-desktop.org/regolith.key`.

**Comment by kgilmer** [comment]
Thanks @samstokes , I'll take a look

**Comment by kgilmer** [comment]
The forums whispered "restart the webserver" and so that's what I did and it seems to be working now for me.  @samstokes can you confirm?

**Reply by samstokes** [reply]
Yes, seems like that fixed it - thanks for the speedy response! My `apt install` succeeded this time.

**Comment by JDKoder** [comment]
@kgilmer 
I am getting a similar error today with the following url:
`Err:5 https://regolith-desktop.org/release-debian-bullseye-amd64 bullseye InRelease
  Certificate verification failed: The certificate is NOT trusted. The certificate chain uses expired certificate.  Could not handshake: Error in the certificate verification. [IP: 104.238.213.207 443]
`
Same condition in browser on the regolith.key url

**Reply by simonfogliato** [reply]
I get the same thing. https://regolith-desktop.com/regolith.key does not exist at the moment.

**Comment by kgilmer** [comment]
Needed to update the cron job to bounce the http server after cert update.  Should be fixed now.  Hopefully doesn't resurface 6 months from now :crossed_fingers:
